modules/extension/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBFunction.java

```diff
@@ -35,7 +35,10 @@
 import org.opengis.filter.expression.Function;
 
 /**
- * MBFunction json wrapper, allowing conversion to a GeoTools Expression.
+ * MBFunction json wrapper, allowing conversion of function to a GeoTools Expression.
+ *
+ * <p>As of MapBox Style Specification v0.41.0, property expressions are the preferred approach for
```

Comment:
"property expressions" is not a term referenced anywhere in the MBStyle spec (I think the term you are looking for is `data expressions` - https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/#data-expressions ), and you've referred to them as just "expressions" elsewhere.

Please stick with a consistent name. The distinction between GeoTools Expressions and MapBox expressions is already confusing enough without making up terms.

Reply:
Okay so `data expression` for mapbox style, and `GeoTools Expression` elsewhere.

Reply:
The https://docs.mapbox.com/mapbox-gl-js/style-spec/other/#function page uses:

> As of v0.41.0, property expressions is the preferred method for styling features based on zoom level or the feature's properties. Zoom and property functions are still supported, but will be phased out in a future release.

Reply:
I had a run at consistently using:
- "data expression"
- "GeoTools Expression"
- "name" (rather than tag or propertyName)
- "value" for object vs "element" for array